<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>User</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Number</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Start Feedback</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QListWidget" name="listWidget">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>EMS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>REAL</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Actions</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="buttonTest">
        <property name="text">
         <string>Start Test Mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCalibrate">
        <property name="text">
         <string>Start Calibration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonStudy1">
        <property name="text">
         <string>Start Study 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonStudy2">
        <property name="text">
         <string>Start Study 2</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
